UsageStatsLoader.inc.php

```diff
@@ -627,6 +674,15 @@ protected function getExpectedPageAndOp() {
 				);
 				$pageAndOp[Application::getContextAssocType()][] = 'catalog/index';
 				break;
+			case 'pps':
+				$pageAndOp = $pageAndOp + array(
+					ASSOC_TYPE_SUBMISSION_FILE => array(
+						'preprint/download'),
+					ASSOC_TYPE_ARTICLE => array(
```

Comment:
(You can use `ASSOC_TYPE_SUBMISSION` as the "abstract case" for app-specific constant `ASSOC_TYPE_ARTICLE`)

Reply:
Check. I have tried to prevent replacing those constants for now in PPS branch so that the cherry-picks would work as well as possible. We can rename those and the filenames after creating the repo

Reply:
How about I change uses of `ASSOC_TYPE_ARTICLE` to `ASSOC_TYPE_SUBMISSION` in the OJS repo? That way your next rebase would fix this, and both apps would use neutral language.

Reply:
Why not!